src/Uno.Toolkit.UI/Behaviors/ItemsRepeaterExtensions.cs

```diff
@@ -184,6 +185,10 @@ private static void OnItemsRepeaterCommandTapped(object sender, TappedRoutedEven
 
 	// Unlike ToggleButton (or Chip which derives from), SelectorItem is not normally selectable on click, unless nested under a Selector.
 
+#if HAS_UNO_WINUI
+	private static ItemsRepeater? _ir;
+#endif
```

Comment:
a static reference should never be used in an extension for its target
since the extension can be used more than one times